typedb-src/modules/ROOT/pages/overview.adoc

```diff
@@ -2,25 +2,25 @@
 :keywords: typedb, typeql, documentation, overview, introduction
 :longTailKeywords: documentation overview, learn typedb, learn typeql, typedb schema, typedb data model
 :pageTitle: Documentation overview
-:summary: A birds-eye view of TypeQL and TypeDB
+:summary: A bird's-eye view of TypeQL and TypeDB
 
-This is the TypeDB server documentation section. It has the following sections:
+TypeDB server documentationm, which contains the following subsections:
 
-* xref:introduction.adoc[] -- a brief description of TypeDB capabilities
+* xref:introduction.adoc[] -- brief description of TypeDB's capabilities
 * xref:installation.adoc[] -- TypeDB server installation manual
-* xref:quickstart.adoc[] -- a guide to quickly set up a TypeDB database
+* xref:quickstart.adoc[] -- guide to quickly set up a TypeDB database
 
 * *Fundamentals*
-** xref:fundamentals/types.adoc[] -- essential information on types and inheritance
-** xref:fundamentals/queries.adoc[] -- essential information on queries
-** xref:fundamentals/patterns.adoc[] -- essential information on pattern matching
-** xref:fundamentals/inference.adoc[] -- essential information on rules and reasoning (inference)
+** xref:fundamentals/types.adoc[] -- types and inheritance
+** xref:fundamentals/queries.adoc[] -- queries and clauses
+** xref:fundamentals/patterns.adoc[] -- query patterns
+** xref:fundamentals/inference.adoc[] -- rules and reasoning
```

Comment:
```suggestion
** xref:fundamentals/types.adoc[] -- types and inheritance
** xref:fundamentals/queries.adoc[] -- queries and clauses
** xref:fundamentals/patterns.adoc[] -- patterns and variables
** xref:fundamentals/inference.adoc[] -- inference and rules
```

Reply:
Worth noting here that all the links are rendered with the respective page's name. E.g., `types.adoc[]` will be rendered as [Type system](https://github.com/vaticle/typedb-docs/pull/690/fundamentals/types.html)
I was trying to avoid repeating the page name. And without a description, those pages would look out of the ordinary in this list.
I think we can reimagine the Overview pages with a slightly different approach, separating a single list into one list per subsection. That would additionally increase the in-page navigation by providing headings for top-level subsections. This idea was suggested by Christoph. I can see how that could be better, especially given not the best design of our multi-level bullet point lists.

Reply:
Good idea.